app/scripts/models/auth_brokers/base.js

```diff
@@ -406,5 +406,8 @@ define(function (require, exports, module) {
     SearchParamMixin
   );
 
-  module.exports = BaseAuthenticationBroker;
+  module.exports = {
+    Constructor: BaseAuthenticationBroker,
+    options: [ 'relier', 'window' ]
```

Comment:
Is it possible to declare `options` on the Constructor or its prototype instead? It seems to me adding the intermediate structure doesn't provide tons of value and seems awfully similar to the prototype chain in other auth brokers, but I could be convinced otherwise. Any advantages to doing it either way?

Declaring options on the constructor's prototype means you wouldn't need to update modules like `fx-desktop-v1.js` that declare no options of its own since options would be available on the prototype chain.